### DISCUSSION TITLE
Billing for Turkey

### DISCUSSION THREAD

**Discussion by sonmez-baris** [community/community#41027] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi,

I want to use my company informations for shopping on Github and take bill for my company. I cannot use Github's invoice as official. Because of there is no my tax number on its. How should i go about it?

Thanks.

**Comment by ossanna16** [comment, marked as answer]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by Southsider13** [reply]
- - [ ] ______

**Comment by Tamarwish** [comment]
Hello
